fenix/messaging.fml.yaml

```diff
@@ -70,6 +70,11 @@ features:
             DEVICE_IOS: os == 'iOS'
             ALWAYS: "true"
             NEVER: "false"
+            DAY_1_AFTER_INSTALL: days_since_install == 1
+            DAY_2_AFTER_INSTALL: days_since_install == 2
+            DAY_3_AFTER_INSTALL: days_since_install == 3
+            DAY_4_AFTER_INSTALL: days_since_install == 4
+            DAY_5_AFTER_INSTALL: days_since_install == 5
```

Comment:
are these required for this issue or is this for future use?

Reply:
Future use.